src/bitcoin-cli.cpp

```diff
@@ -349,6 +350,62 @@ class NetinfoRequestHandler : public BaseRequestHandler
         const double milliseconds{round(1000 * seconds)};
         return milliseconds > 999999 ? "-" : ToString(milliseconds);
     }
+    const UniValue NetinfoHelp()
+    {
+        return std::string{
+            "-netinfo level \"help\" \n\n"
```

Comment:
Wouldn't this be better summarized as `-netlinfo level|"help"` instead of a separate argument?

I've also noticed that it doesn't error out on invalid values:
```
$ src/bitcoin-cli -regtest -netinfo help
[help text]
$ src/bitcoin-cli -regtest -netinfo 0
Bitcoin Core v21.99.0-cd42490f045aa2c3454d0c5518199119f5d1ea7d regtest - 70016/Satoshi:21.99.0/

        ipv4    ipv6   onion   total  block-relay
in         0       0       0       0       0
out        0       0       0       0       0
total      0       0       0       0       0

Local addresses: n/a
$ src/bitcoin-cli -regtest -netinfo hel
Bitcoin Core v21.99.0-cd42490f045aa2c3454d0c5518199119f5d1ea7d regtest - 70016/Satoshi:21.99.0/

        ipv4    ipv6   onion   total  block-relay
in         0       0       0       0       0
out        0       0       0       0       0
total      0       0       0       0       0

Local addresses: n/a
```

Reply:
done